frontend/app/controllers/create-badges.js

```diff
@@ -616,9 +616,6 @@ export default Controller.extend({
 
     },
 
-    defaultlogoimage() {
-    },
```

Comment:
Maybe comment it out as that is a `should make one test for`

Reply:
Sir the error was of a duplicate key as the defaultlogoimage() function already exists in create-badge.js so should i change the function name to defaultLogoImage()?